### PR TITLE
3d line of sight wibble fix

### PR DESCRIPTION
**Pull request by spartahawk**
When using the `line_of_sight_3d()` function, explosions from meteors (and likely grenades if they explode at just the right time) were occurring within "wibble" bumps on floors and walls, which were handled by the line of sight function as solid materials, blocking area damage. This attempts to resolve the issue.

### CODE REVIEW

File: src/creature_senses.c [hunk 1]
```diff
@@ -744,11 +758,19 @@ TbBool line_of_sight_3d(const struct Coord3d *frpos, const struct Coord3d *topos
         if (point_in_map_is_solid(&nextpos)) {
             SYNCDBG(7, "Player cannot see through (%d,%d) due to linear path solid flags (downcount %d)",
                 (int)nextpos.x.stl.num,(int)nextpos.y.stl.num,(int)distance);
+
+            JUSTMSG("Player cannot see through (%d,%d) due to linear path solid flags (downcount %d)",
```
Comment (Loobinex) [inline src/creature_senses.c]: I'll test with this in for now, but we shouldn't forget to remove this before we merge.
Just like the one a few lines below

Comment (spartahawk) [reply src/creature_senses.c]: Right.

**Comment by Loobinex** [comment]
I see nothing wrong with it, I can consistently do damage when I shoot walls and floors.
@Schizz, do you think this works ok? Also with your grenades?

**Comment by Schizz** [comment]
I believe my grenades are OK with it.
As long as explosions work my grenades should work too. :)

**Comment by spartahawk** [comment]
That last commit "Merge branch ..." was just me merging your commit with mine locally on my machine after I had already changed a few things and made my own local commit before pushing. It's nothing.

**Pull request merged**